### PR TITLE
adding links to personal pages

### PR DESCRIPTION
**Pull request by kemalayhan**

### CODE REVIEW

File: pages/_slug.vue [hunk 1]
```diff
@@ -1,6 +1,14 @@
 <template>
   <div class="container">
-    {{ doc.name }}
+    <ul>
+        <li>{{ doc.name }}</li>
+        <li><a :href="doc.twitter_handle ">Twitter</a></li>
```
Comment (cagataycali) [inline pages/_slug.vue:5]: `doc.twitter_handle` has whitespace before closing

File: pages/_slug.vue [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+        <li><a :href="doc.github ">Github</a></li>
+        <li><a :href="doc.linkedin ">Linkedin</a></li>
+        <li>Interests: {{doc.interests}}</li>
+        <li>Goals: {{doc.goals}}</li>
```
Comment (cagataycali) [inline pages/_slug.vue:9]: `<li v-if='doc.goals'>Goals: {{doc.goals}}</li>`

**Comment by kemalayhan** [comment]
Is last commit true? 
v-if='doc.goals && doc.goals.length' --> for this statement: when user don't write goals it comes empty string. So goals exists and length equals to 1. Because of this i use length > 1

**Comment by cagataycali** [comment]
You're right! My fault, thx!

**Comment by kemalayhan** [comment]
I've always checked a specific user. But I realized he was putting on a space character. So the user length of goals is 1. As a result when user doesn't write anything length equals to 0.

**Pull request merged**